src/validator-map.js

```diff
@@ -55,21 +55,26 @@ VALIDATOR_MAP[DATA_TYPES.TIME] = Utils.buildRegexCheck('isTime');
 // numbers:
 // 1, 2, 3, +40, 15,121
 const intRegexCheck = Utils.buildRegexCheck('isInt');
-VALIDATOR_MAP[DATA_TYPES.INT] = function isInt(value) {
+function isInt(value) {
   if (intRegexCheck(value)) {
     var asNum = parseInt(value.toString().replace(/(\+|,)/g, ''), 10);
     return asNum > Number.MIN_SAFE_INTEGER && asNum < Number.MAX_SAFE_INTEGER;
   }
 
   return false;
 };
+VALIDATOR_MAP[DATA_TYPES.INT] = isInt;
 
 // 1.1, 2.2, 3.3
-VALIDATOR_MAP[DATA_TYPES.FLOAT] = Utils.buildRegexCheck('isFloat');
+const floatRegexCheck = Utils.buildRegexCheck('isFloat');
```

Comment:
why not edit the float regex to make `.` optional?

Reply:
With that approach, long integer values would be considered floats